client/src/components/footer/index.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import "./style.css";
+import { Icon } from "antd";
+
+export const Footer = () => {
+  const socialLinks = {
+    FB: "https://www.facebook.com/socialstreetsco",
+    instagram: "https://www.instagram.com/socialstreetsco/",
+    twitter: "https://twitter.com/socialstreetsco",
+    linkedIn: "https://www.linkedin.com/company/10668755/"
+  };
+  return (
+    <footer className="footer">
+      <a
+        className="footer--a"
```

Comment:
If you're trying to use BEM as a naming methodology for the classes you should change the class name for the a tag to be an element.
```suggestion
        className="footer__anchor"
```
or
```suggestion
        className="footer__a"
```
For me, I prefer the first one because it's a bit clearer to understand.
Read [This doc](http://getbem.com/introduction/) and if you'd like we could discuss it to use the other one [here](https://en.bem.info/).